client/src/views/home/HomePage.vue

```diff
@@ -299,7 +299,7 @@ async function openCreateOrganizationModal(bootstrapLink?: string, defaultServer
     component: CreateOrganizationModal,
     canDismiss: true,
     cssClass: 'create-organization-modal',
-    backdropDismiss: isLargeDisplay.value ? false : true,
+    backdropDismiss: false,
     showBackdrop: true,
     breakpoints: isLargeDisplay.value ? undefined : [0, 0.5, 1],
```

Comment:
Remove the breakpoint `0` as well, otherwise the window can be closed that way without confirmation.